App.UserSupport/Repositories/InMemoryUserSupportRepository.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using System.Collections.Generic;
 using App.Configuration;
 using App.UserSupport.Models;
+using System.Linq;
 
 namespace App.UserSupport.Repositories
 {
     public interface IHandlingsRepository
```

Comment:
it is better to move this interface from this file, because file name is "InMemoryUSerSupportRepository"

Reply:
Ok, i'll fix it